tools/user_management/main.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+require 'yaml'
+require 'rest-client'
+
+require_relative './lib/actions'
+require_relative './lib/user'
+
+UAA_API_URL = ENV.fetch('UAA_API_URL')
+CF_TOKEN    = ENV.fetch('CF_TOKEN')
+USERS_CONFIG_PATH = ENV.fetch('USERS_CONFIG_PATH')
+
+uaa_client = RestClient::Resource.new(
+  UAA_API_URL,
+  headers: {
+    'Authorization' => CF_TOKEN,
+    'Content-Type' => 'application/json'
+  }
+)
+users_config = YAML.safe_load(File.read(USERS_CONFIG_PATH)).map { |uo| User.new(uo) }
```

Comment:
I've discovered a bug. 😮 We pass `deploy_envs` from the YAML file onto the `User` class… but we never use it. We need to filter users down to those present in the current `DEPLOY_ENV`. 🤦‍♀